handlers.go

```diff
@@ -51,22 +51,65 @@ func (s *externalCoordinatorServer) RegisterMissionControl(ctx context.Context,
 			stalePairsRemoved)
 	}
 
+	// Initialize a map to aggregate mission control data.
```

Comment:
Nit: The  `RegisterMissionControl` can maybe get a more detailed comment what it does specifically and also that it updates existing data if new data is registered compared by the most recent timestamp.

Reply:
Thanks! updated